routers/rooms.js

```diff
@@ -820,6 +1100,34 @@ async function hasPermission(user_id, room_id, permission) {
     return role[permission];
 }
 
+/**
```

Comment:
Function for logging an audit event on a room.
Also my first time using JSDoc, extremely useful tool.